### PR TITLE
add view command, currently renders under the search bar. test cases …

### PR DESCRIPTION
**Pull request by zhengyup**
Add view command, currently renders like this 

Screenshot 2023-10-16 at 3 01 43 PM

will write the test cases after merging timi's PR, pushing so that youwei can settle the rendering of the UI first

### CODE REVIEW

**Comment by zhengyup** [comment]
There are still references to person since i havent merged tim's PR. will change afterwards

File: src/main/java/seedu/letsgethired/logic/commands/ViewCommand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+/**
+ * Selects and displays an Application identified using it's displayed index from the intern tracker.
+ */
+public class ViewCommand extends Command {
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Selects and displays the intern application identified by the index number used in the "
+            + "displayed "
+            + "person list.\n"
```
Comment (tim-pipi) [inline src/main/java/seedu/letsgethired/logic/commands/ViewCommand.java:23]: Perhaps this should be intern application list instead?

File: src/main/java/seedu/letsgethired/logic/commands/ViewCommand.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_INTERN_APPLICATION_SUCCESS = "View application: %1$s";
```
Comment (tim-pipi) [inline src/main/java/seedu/letsgethired/logic/commands/ViewCommand.java:27]: Perhaps we could change this to `Viewed intern application` to follow the convention of the other commands

File: src/main/java/seedu/letsgethired/model/InternTracker.java [hunk 3]
```diff
@@ -17,6 +17,8 @@ public class InternTracker implements ReadOnlyInternTracker {
 
     private final UniqueApplicationList internApplications;
 
+    private InternApplication currentApplication;
```
Comment (tim-pipi) [inline src/main/java/seedu/letsgethired/model/InternTracker.java:20]: Maybe `selectedInternApplication` would be less ambiguous?

File: src/main/java/seedu/letsgethired/model/InternTracker.java [hunk 4]
```diff
@@ -110,6 +112,14 @@ public ObservableList<InternApplication> getApplicationList() {
         return internApplications.asUnmodifiableObservableList();
     }
 
+    public void setCurrentApplication(InternApplication currentApplication) {
```
Comment (tim-pipi) [inline src/main/java/seedu/letsgethired/model/InternTracker.java:115]: Same here and other instances of `currentApplication` - perhaps using `selectedInternApplication` would be more consistent with the rest of the code and less ambigous!

**Review by tim-pipi — COMMENTED** [review]
LGTM! Just some nits to pick with respect to the naming of the field `currentApplication`! 

Also, I think it might be beneficial to include some test cases, to ensure that you get the code coverage! I think the "Add Remark Command" tutorial has some stuff regarding to this too!

**Review by tim-pipi — APPROVED** [review]
LGTM! Thanks for the changes and the test cases!

**Review by tyouwei — APPROVED** [review]
LGTM on my side as well. Merging the PR in.

**Pull request merged**